docs/EDTT_framework_Babblesim.md

```diff
@@ -110,6 +110,64 @@ All requests and replies share a common syntax of the form:
 <request_id>, <request_size>, <reply_id> and <reply_size> are little-endian 16 bit unsigned numbers. <request_parameter> and <reply_parameter> are 8 bit numbers.
 ```
 
+## Packet inspection
+
+It is possible to inspect the raw packets sent by the upper and lower tester via BabbleSims dump files. Note that you will usuallly want to use the `-dump_imm` command line argument when using packet inspection; Otherwise the dump files will use cached writes and the latest packets will likely not show up in EDTT.
+
+The raw packets will get decoded into a `Packet` before being handed over to the test cases via the `Packets` class. The content of a `Packet` is:
+
+* `direction`: Either `'Tx'` or `'Rx'`. Note that the current implementation only reads the Tx part of the dump files, so this will currently always be `'Tx'`
+* `idx`: The BSim device id of the device that received or transmitted the package (see `direction` for whether it was a transmit or receive). Will correesppnd to lower or upper tester
+* `ts`: The start timestamp of the packet (in microseconds)
+* `aa`: The access address used
+* `channel_num`: The channel number used (note: Not the channel *index*, but the channel *number* - ie. advertising channels are 0, 12 and 39)
+* `phy`: The BLE PHY used - one of `'1M'` or `'2M'`
+* `data`: The raw data of the packet (excluding header)
+* `type`: The type of packet, for example `'ADV_IND'` or `'CONNECT_IND'`
+* `header`: The header of the packet - a named tuple containing `pdu_type`, `ch_sel`, `tx_add`, `rx_add` and `payload_len`
+* `payload`: The decoded payload of the packet. Content depends on the type of packet; It will generally be a named tuple containing the fields specified in the BT Core Spec. One notable exception is for the extended advertisement packets - for these the payload is a `dict` since the fields may or may not be present
+
+### Packets class interface
+
+There are a few different ways to inquire about packets sent on the phy from within a test case. Generally, the way to use the API is to filter for the packets you are interested - see the API describtion below.
+
+#### Packets.fetch(packet_filter=())
```

Comment:
Does calling fetch get the set of packets updated? (meaning it goes to the files to look for more?)

Reply:
Yes it reads automagically from the files :-)

Reply:
Ok, it would be nice to mention it in the doc :)